cEP-0022.md

```diff
@@ -0,0 +1,145 @@
+# Next Gen Bear Design
+
+| Metadata |                                              |
+| -------- | ---------------------------------------------|
+| cEP      | 22                                           |
+| Version  | 1.0                                          |
+| Title    | Quickstart Green Mode                        | 
+| Authors  | Ishan Srivastava <mailto:ishan.srt@gmail.com>|
```

Comment:
Give a space in the end otherwise gitmate will be mad when it woke up.
`<mailto:ishan.srt@gmail.com>|` --> `<mailto:ishan.srt@gmail.com> |`